main.cpp

```diff
@@ -1010,13 +730,371 @@ Part 1e - Step 19: Request a review
+    std::string material = "Spruce Wood";
+
+    struct Strings
+    {
+        float lowEStringGauge = .040f; 
 
+        float dStringGauge = .010f;
+        std::string brand = "Ernie Ball";
+        std::string sku = "2229";
+        std::string material ="Nickel";
+        
+        void bend(std::string whichString);
+        void slide(std::string targetNote);
+        void snap(std::string whichString);
+    
+    };
+    // 3 things it can do:
+    //     1) Play individual notes
+    void playNote(Strings strings, std::string whichNote);
+    //     2) Be tuned to different tunings
+    void tuneString(Strings strings,std::string whichString);
+    //     3) Strum notes
+    void strumNotes (Strings strings, std::string whichChord);
+};
+
+struct Wings
+{
+    // 5 properties:
+    //     1) Wing shape (std::string)
+    std::string shape = "Airfoil";
+    //     2) Wing length (int)
+    int length = 130;
+    //     3) Ratio of wings length to width (int)
+    int ratioLengthToWidth = 55;
+    //     4) Angle of wing tilt (float)
+    float tiltAngle = 20.f;
+    //     5) Flap angle (float)
+    float flapAngle = 30.f;
+    // 3 things it can do:
+    //     1) Generate lift
+    void generateLift();
+    //     2) Provide stability
+    void stabilize();
+    //     3) Adjust to various conditions and scenarios
+    void adjustToTurbulance();
+}; 
+
+struct Engines 
+{
+//     5 properties:
+//         1) Number of engines (int)
+    int numEngines = 2;
+//         2) Power output (float)
+    float powerOutput = 30.f;
+//         3) Fuel efficiency (float)
+    float fuelEfficiency = 10.f;
+//         4) Emissions (float)
+    float emissions = 20.f;
+//         5) Size (int)
+    int size = 20;
+
+    struct Turbines
+    {
+        int lowPressureLenth = 84; 
+        int highPressureWidth = 93;
+        std::string brand = "GE";
+        std::string model = "CF3942";
+        std::string material ="Titanium";
+
+        void thrust(std::string whichTurbine);
+        void efficiency(std::string targetSpeed);
+        void noiseLevel(std::string whichTurbine);
+    };
 
 
+//     3 things it can do:
+//         1) Provide thrust
+    void engageThrust(Turbines turbines);
+//         2) Generate elecricity
+    void increasePower(Turbines turbines, int amountOfIncrease);
+//         3) Control speed and altitude
+    void controlSpeedAndAltitude(Turbines turbines, int amountOfChange);
+
+    Turbines engaged;
+};
+
+struct CargoSpace
+{
+    // 5 properties:
+    //     1) Volume (float)
+    float volume = 64.f;
+    //     2) Weight (int)
+    int weight = 1300;
+    //     3) Temperature (int)
+    int temperature = 75;
+    //     4) Capacity (int)
+    int capacity = 400;
+    //     5) Loading time (float)
+    float loadingTime = 45;
+    // 3 things it can do:
+    //     1) Transport
+    void transportCargo();
+    //     2) Store
+    void storeCargo();
+    //     3) Secure
+    void secureCargo();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise